data/publiccloud/terraform/azure.tf

```
@@ -108,6 +113,22 @@ resource "azurerm_resource_group" "openqa-group" {
   }, var.tags)
 }
 
+resource "azurerm_virtual_network" "openqa-network" {
+  count               = var.subnet_id == "" ? 1 : 0
+  name                = "${azurerm_resource_group.openqa-group.name}-vnet"
+  address_space       = ["10.0.0.0/16"]
```

```suggestion
  address_space       = ["10.0.118.0/24"]
```
Do we really need/want a `/16` subnet? I think a `/24` with a not-typically used subnet can avoid some nasty collisions.

---

`address_prefixes` below would need to be adapted accordingly.

---

Thank you for raising your understandable concerns.

From my understanding, vnets with the same address space are distinct between two resource groups on Azure. Unfortunately I could not find official documentation for this assumption of which I am quite sure.
On this foundation, a conflict could not happen in this case, because each test spawns its own resource group and the vnet is created within this separate resource group.

---

They are indeed isolated, but a `/16` subnet is still wasteful, I'd even say "luxuriously wasteful" :wink: